lib/stdlib/doc/src/timer.xml

```diff
@@ -69,6 +69,31 @@
   </datatypes>
 
   <funcs>
+    <func>
+      <name name="apply_after" arity="2" since=""/>
+      <fsummary>Spawn a process evaluating <c>erlang:apply(Function, [])</c>
+        after a specified <c>Time</c>.</fsummary>
+      <desc>
+        <p>Evaluates <c>spawn(erlang, apply, [<anno>Function</anno>,
+          []])</c> after <c><anno>Time</anno></c> milliseconds.</p>
+        <p>Returns <c>{ok, <anno>TRef</anno>}</c> or
+          <c>{error, <anno>Reason</anno>}</c>.</p>
+      </desc>
+    </func>
+
+    <func>
+      <name name="apply_after" arity="3" since=""/>
+      <fsummary>Spawn a process evaluating <c>erlang:apply(Function, Arguments)</c>
+        after a specified <c>Time</c>.</fsummary>
```

Comment:
I don't think we should have this function (nor the other ones that take both a fun and arguments). We don't use this style for the `lists` module or for `spawn`, so I don't think we should use it here.

(I see that `timer:tc()` since OTP 26.0 has variants that take both a fun and arguments. Being a family functions that is often use interactively, there is some justification for having extra convenience functions.)